analytics/analytics.go

```diff
@@ -22,6 +22,13 @@ const (
 	PredefinedTagGraphAnalytics = "tyk-graph-analytics"
 )
 
+const (
```

Comment:
It might make sense for pump to have its own version, but I just wanted to let you know that this exact thing exists in graphql-go-tools.